test/test_utils.jl

```diff
@@ -0,0 +1,15 @@
+# Create a default kernel from two parameters k[1] and k[2]
+make_kernel(k) = softplus(k[1]) * (SqExponentialKernel() ∘ ScaleTransform(softplus(k[2])))
+
+# Computes the optimal closed form solution for the variational posterior
+# q(u) (e.g. # https://krasserm.github.io/2020/12/12/gaussian-processes-sparse/
+# equations (11) & (12)). Assumes a ZeroMean function.
+function exact_variational_posterior(fu, fx, y)
```

Comment:
Could we encode the assumption in code, e.g.
```suggestion
# equations (11) & (12)).
function exact_variational_posterior(fu, fx, y)
    @assert fu.f.mean isa ZeroMean
```
?

Reply:
`@assert` is generally not a good (check its docs). It's better to have something like `fu.f.mean isa ZeroMean || error("Nope")`